api/src/models/lng/warning-lng.js

```diff
@@ -6,6 +6,7 @@ module.exports = require('../../utils/model-schema-generator')(
     _schemaName: { type: String, default: 'WarningLNG' },
     _epicProjectId: { type: 'ObjectId', default: null, index: true },
     _sourceRefId: { type: 'ObjectId', default: null, index: true },
+    _sourceRefOgcWarningId: { type: String, default: null },
```

Comment:
Is there a reason why we have another field for this? Is it possible to use _sourceRefId?

Reply:
`_sourceRefId` is `ObjectId` and the ref id here is a `string`

Reply:
yeah the incoming field used as a reference is a string `Title`